lms/templates/courseware/courseware-chromeless.html

```diff
@@ -201,4 +201,138 @@
+      this.onMouseUp = this.stopResize.bind(this);
+
+      this.init();
+    }
+
+    /**
+     * Binds the mousedown listener on the handle in capture phase, 
+     * preventing AnnotatorJS from receiving the event first.
+     */
+    init() {
+      this.draggable.addEventListener(
+        'mousedown',
+        (event) => {
+          event.stopImmediatePropagation();
+          this.startResize(event);
+        },
+        { capture: true }
+      );
+    }
+
+    /**
+     * Called on mousedown. Stores the initial position and size, then 
+     * starts listening for mousemove/mouseup at document level.
+     * 
+     * @param {MouseEvent} event - The mousedown event.
+     */
+    startResize(event) {
+      this.startX = event.clientX;
+      this.startY = event.clientY;
+      this.startWidth = this.textarea.offsetWidth;
+      this.startHeight = this.textarea.offsetHeight;
+
+      document.addEventListener('mousemove', this.onMouseMove);
+      document.addEventListener('mouseup', this.onMouseUp);
+    }
+
+    /**
+     * Handles mousemove events, updating the <textarea> width/height 
+     * based on the difference between current and initial pointer positions.
+     * 
+     * @param {MouseEvent} event - The mousemove event.
+     */
+    onResize(event) {
+      const deltaX = event.clientX - this.startX;
+      const deltaY = event.clientY - this.startY;
+
+      const editor = this.draggable.closest('.annotator-editor');  
+      const newHeight = editor.classList.contains('annotator-invert-y')
+        ? this.startHeight + deltaY
+        : this.startHeight - deltaY;
+      const newWidth = this.startWidth + deltaX;
+
+      this.textarea.style.setProperty('width',  newWidth + 'px',  'important');
+      this.textarea.style.setProperty('height', newHeight + 'px', 'important');
+    };
+
+    /**
+     * Called on mouseup. Stops listening to mousemove/mouseup 
+     * and finalizes the resize action.
+     */
+    stopResize() {
+      document.removeEventListener('mousemove', this.onMouseMove);
+      document.removeEventListener('mouseup', this.onMouseUp);
+    };
+  }
+
+  /**
+   * Given a .annotator-resize element, it finds the target <textarea>,
+   * and creates a ResizableTextarea instance. It also starts listening
+   * for click events on the annotator's cancel button to restablish
+   * the textarea to its default size after resizing.
+   * 
+   * 
+   * @param {HTMLElement} resizer - The element with class .annotator-resize
+   */
+   function handleResizer(resizer) {
+    const editor = resizer.closest('.annotator-editor');
+    const textarea = editor.querySelector('textarea');
+
+    if (!textarea) return;
+    new ResizableTextarea(resizer, textarea);
+    document.addEventListener('click', (event) => {
+      if (event.target.matches('.annotator-cancel')) {
+        textarea.style.removeProperty('width');
+        textarea.style.removeProperty('height');
+      };
+    });
```

Comment:
I think this can be moved to the init function of ResizableTextarea. In addition, you should remove the listener otherwise every time that ".annotator-resize" changes, you will be stacking new click events. @Serafin-dev